htdocs/product/stock/class/api_warehouses.class.php

```diff
@@ -275,6 +276,60 @@ public function delete($id)
 		);
 	}
 
+	/**
+	 * List Product in warehouses
+	 *
+	 * Get a list of product in a warehouse
+	 *
+	 * @param string	$id	warehouse ID
+	 * @param int		$limit		Limit for list
+	 * @return array    Array of product in warehouse
```

Comment:
You can add here :
```
	 * @phan-return array<array{rowid:int,ref:string,fk_product_type:int,label:string,price:float,price_ttc:float,entity:int,pmp:float,reel:float,reelpmp:float,reelprice:pmp}>
	 * @phpstan-return array<array{rowid:int,ref:string,fk_product_type:int,label:string,price:float,price_ttc:float,entity:int,pmp:float,reel:float,reelpmp:float,reelprice:pmp}>
```
to avoid the 2 next CI warning